org.code.toboggan.ui/src/org/code/toboggan/ui/dialogs/extensions/DialogWSEvent.java

```diff
@@ -14,8 +20,25 @@
 import org.eclipse.swt.widgets.Display;
 import org.eclipse.swt.widgets.Shell;
 
+import clientcore.dataMgmt.SessionStorage;
+
 public class DialogWSEvent implements IWSEvent {
 	private Logger logger = LogManager.getLogger(this.getClass());
+	
+	public DialogWSEvent(){
+		CoreActivator.getSessionStorage().addPropertyChangeListener(new PropertyChangeListener() {
+			
+			@Override
+			public void propertyChange(PropertyChangeEvent evt) {
+				if (evt.getPropertyName().equals(SessionStorage.USERNAME)){
+					Display.getDefault().asyncExec(() -> {
+						Shell shell = Display.getDefault().getActiveShell();
+						new ConfirmSubscribeDialog(shell).open();
+					});
+				}				
+			}
+		});
```

Comment:
Hmm. My personal view is that this also is a websocket event, since we're updating the authentication information (it's basically a proxy to listen for websocket auth). It also can't be in the Core plugin, since it relies on dialogs, and putting it in the UI activator fires it for all the tests as well, which also should not happen.